test/kixi/data_acquisition/unit/db/inmemory_test.clj

```diff
@@ -0,0 +1,53 @@
+(ns kixi.data-acquisition.unit.db.inmemory-test
+  (:require [clojure.test :refer :all]
+            [com.stuartsierra.component :as component]
+            [kixi.data-acquisition.db :as db]
+            [kixi.data-acquisition.db.inmemory :refer :all]))
+
+(def db (atom nil))
+
+(defn create-db
+  [all-tests]
+  (let [system
+        (component/start-system
+         (component/system-map
+          :db (map->DbInMemory {})))]
+    (reset! db (:db system))
+    (all-tests)
+    (component/stop-system system)
+    (reset! db nil)))
+
+(use-fixtures :each create-db)
+
+(deftest create-table-no-index
```

Comment:
most of the tests in here look like they could be described as fdef generative tests. Might be worth doing that for future iterations.